src/org/loklak/api/cms/DumpDownloadServlet.java

```diff
@@ -47,28 +46,31 @@
     private static final long serialVersionUID = 2839106194602799989L;
 
     @Override
-    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    protected void doPost(HttpServletRequest request,
+        HttpServletResponse response) throws ServletException, IOException {
         doGet(request, response);
     }
-    
+
     @Override
-    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-        Query post = RemoteAccess.evaluate(request);
+    protected void doGet(HttpServletRequest request,
+        HttpServletResponse response) throws ServletException, IOException {
+        final Query post = RemoteAccess.evaluate(request);
         String path = request.getPathInfo();
-        long now = System.currentTimeMillis();
-        
-        int limited_count = (int) DAO.getConfig("download.limited.count", (long) Integer.MAX_VALUE);
-        String limited_message = DAO.getConfig("download.limited.message", "");
-        
+        final long now = System.currentTimeMillis();
+
+        int limitedCount = (int) DAO.getConfig("download.limited.count", (long) Integer.MAX_VALUE);
+        String limitedMessage = DAO.getConfig("download.limited.message", "");
+
         if (path.length() <= 1) {
             // send directory as html
 
             FileHandler.setCaching(response, 60);
             response.setContentType("text/html");
             response.setCharacterEncoding("UTF-8");
             response.setStatus(HttpServletResponse.SC_OK);
-            
-            OutputStreamWriter osw = new OutputStreamWriter(response.getOutputStream(), StandardCharsets.UTF_8);
+
+            OutputStreamWriter osw = new OutputStreamWriter(response.getOutputStream(),
+                StandardCharsets.UTF_8);

```

Comment:
Please don't break this into a new line. Add checkstyle exception if needed to suppress the warning.